crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -577,47 +1515,54 @@ mod YASPoolTests {
     use yas_core::contracts::yas_router::{
         YASRouter, IYASRouterDispatcher, IYASRouterDispatcherTrait
     };
-    use yas_core::tests::utils::constants::PoolConstants::{TOKEN_A, TOKEN_B, POOL_ADDRESS, WALLET};
+    use yas_core::tests::utils::constants::PoolConstants::{
+        TOKEN_A, TOKEN_B, POOL_ADDRESS, WALLET, encode_price_sqrt_1_1, encode_price_sqrt_1_10
+    };
     use yas_core::tests::utils::constants::FactoryConstants::{
         POOL_CLASS_HASH, FeeAmount, fee_amount, tick_spacing
     };
     use yas_core::contracts::yas_erc20::{
         ERC20, ERC20::ERC20Impl, IERC20Dispatcher, IERC20DispatcherTrait
     };
 
-    fn setup() -> (IYASPoolDispatcher, IERC20Dispatcher, IERC20Dispatcher) {
+
+    fn setup() -> (
+        IYASPoolDispatcher, IERC20Dispatcher, IERC20Dispatcher, IYASRouterDispatcher, i32, i32
+    ) {
         let mint_callback = deploy_mint_callback(); // 0x1
```

Comment:
know it's not part of the scope, but could we rename this to `yas_router` for clarity?

Reply:
the mint_callback? sure thing